assets/src/application/ui/display/espressoIcon/types.ts

```diff
@@ -12,4 +12,12 @@ export enum Icon {
 	REM = 'recurring-date',
 	ROTATE = 'image-rotate',
 	SAVE = 'save',
+	EDIT = 'edit',
+	TICKET_ALT = 'ticket-alt',
```

Comment:
let's just use `TICKET` for the key. We can keep the value as 'ticket-alt' so it's clear which WP dashicon was used

```suggestion
	TICKET = 'ticket-alt',
```